observability/logging/logging-6.0/log6x-meta-contributing.adoc

```diff
@@ -0,0 +1,46 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="log6x-meta-contributing"]
+= Contributing to logging documentation
+:context: logging-6x
+
+[IMPORTANT]
+====
+Do not include this file in the topic map. This is a guide meant for contributors, and is not intended to be published.
+====
+
+Logging consists of the Red Hat Openshift Logging Operator (aka the Cluster Logging Operator), and an accompanying log store Operator. Either the Loki Operator (current/future), or Elasticsearch (deprecated). Either vector (current/future) or fluentd (deprecated) handles log collection and aggregation. Operators use custom resources (CR) to manage applications and their components. High-level configuration and settings are provided by the user within a CR. The Operator translates high-level directives into low-level actions, based on best practices embedded within the Operator’s logic. A custom resource definition (CRD) defines a CR and lists all the configurations available to users of the Operator. Installing an Operator creates the CRDs, which are then used to generate CRs.
+
+== Operator CRs:
+* `Red Hat Openshift Logging Operator`
+** (Deprecated) `ClusterLogging` (CL) - Deploys the collector and forwarder which currently are both implemented by a daemonset running on each node.
+** `ClusterLogForwarder` (CLF) - Generates collector configuration to forward logs per user configuration.
+* `Loki Operator`:
+** `LokiStack` - Controls the Loki cluster as log store and the web proxy with OpenShift Container Platform authentication integration to enforce multi-tenancy.
+** `AlertingRule` - Alerting rules allow you to define alert conditions based on LogQL expressions.
+** `RecordingRule` - Recording rules allow you to precompute frequently needed or computationally expensive expressions and save their result as a new set of time series.
+** `RulerConfig` - The ruler API endpoints require to configure a backend object storage to store the recording rules and alerts.
+* (Deprecated) `OpenShift Elasticsearch Operator` [Note: These CRs are generated and managed by the `ClusterLogging` Operator, manual changes cannot be made without being overwritten by the Operator.]
+** `ElasticSearch` - Configure and deploy an Elasticsearch instance as the default log store.
+** `Kibana` - Configure and deploy Kibana instance to search, query and view logs.
```

Comment:
We do not publish Elasticsearch Operator anymore.

```suggestion
```

Reply:
= Contributing to logging documentation
:context: logging-6x

[IMPORTANT]
====
Do not include this file in the topic map. This is a guide meant for contributors, and is not intended to be published.